sdk/java-sdk-spring/src/it/java/com/example/wiring/views/UserCountersView.java

```diff
@@ -28,20 +28,21 @@
 import kalix.javasdk.annotations.Table;
 import kalix.javasdk.annotations.ViewId;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 
 import java.util.Optional;
 
 @ViewId("user-counters")
 public class UserCountersView {
   @GetMapping("/user-counters/{user_id}")
   @Query(
-      "SELECT users.*, counters.* as counters"
-          + " FROM users"
-          + " JOIN assigned ON assigned.assigneeId = users.id"
-          + " JOIN counters ON assigned.counterId = counters.id"
-          + " WHERE users.id = :user_id"
-          + " ORDER BY counters.id")
-  public UserCounters get(String userId) {
+    "SELECT users.*, counters.* as counters"
+      + " FROM users"
+      + " JOIN assigned ON assigned.assigneeId = users.id"
+      + " JOIN counters ON assigned.counterId = counters.id"
+      + " WHERE users.id = :user_id"
+      + " ORDER BY counters.id")
```

Comment:
For the record, we can use multi-line strings. We have used it in a few other places.